Date: Tue, 17 May 2005 23:54:35 -0400
From: A Costa <agcosta@gis.net>
Reply-To: A Costa <agcosta@gis.net>, 309572@bugs.debian.org
Sender: A Costa <alfie@Arf>
To: Debian Bug Tracking System <submit@bugs.debian.org>
Subject: Bug#309572: 'man cdrwtool' typo: "pktsetup (8)" instead of "pktsetup (1)"

Found a typo in '/usr/share/man/man1/cdrwtool.1.gz', see attached '.diff'.


Reason why it should be "1" and not "8":

Since there's no section #8 man page...

  % dlocate -L udftools | grep "pktsetup.[1-9].gz"
  /usr/share/man/man1/pktsetup.1.gz

...programs like 'pinfo' link to nowhere if you click
on 'pktsetup (8)'.



--- ./doc/cdrwtool.1.orig	2005-10-03 13:11:38.000000000 +0000
+++ ./doc/cdrwtool.1	2005-10-03 13:14:16.000000000 +0000
@@ -140,4 +140,4 @@
 http://linux-udf.sourceforge.net.
 
 .SH "SEE ALSO"
-.BR pktsetup (8)
+.BR pktsetup (1)

--------------------------------------------------------------------------

From: A Costa <agcosta@gis.net>
To: Debian Bug Tracking System <submit@bugs.debian.org>
Subject: Bug#288355: udftools: 'man cdrwtool' should describe the '-q' switch
Date: Mon, 03 Jan 2005 03:47:34 -0500

The '-q' switch is mentioned twice in the 'cdrwtool' man page:
...but under OPTIONS there's no description of what '-q' does.

--- ./doc/cdrwtool.1.orig	2007-11-30 11:25:24.005219775 +0100
+++ ./doc/cdrwtool.1	2007-11-30 11:28:47.516817240 +0100
@@ -83,6 +83,9 @@
 .IP "\fB\-u \fIlength\fP"
 Make a UDF filesystem using \fIlength\fP number of blocks.
 
+.IP "\fB\-q\fP"
+`Quick setup': blank the disc, format it as one large track and write a UDF filesystem.
+
 .IP "\fB\-m \fIoffset\fP"
 Format the disc at \fIoffset\fP number of blocks.
 
--------------------------------------------------------------------------

From: Jari Aalto <jari.aalto@cante.net>
To: Debian Bug Tracking System <submit@bugs.debian.org>
Subject: Bug#372093: udftools: cdrwtool [manual] Mention default settings and clarify options -l
Date: Thu, 08 Jun 2006 11:54:13 +0300

-l -w -p:
The -s option changes these parameters, but the documentation does
not mention what are the default values. Please add word "(default)"
next to the choices, like:
-p type
        Set packet type. Either `0' (default) or `1', corresponding to variable and
        fixed packet sizes respectively.


--- ./doc/cdrwtool.1.orig	2007-11-30 12:19:02.688642090 +0100
+++ ./doc/cdrwtool.1	2007-11-30 12:27:20.216994615 +0100
@@ -101,6 +101,8 @@
 Set write parameters determined by
 .BR "\-l" , " \-w" ", and" " \-p"
 options for the disc.
+.br
+(Note by your Debian maintainer: Apparently this option does nothing.)
 
 .IP "\fB\-v \fIversion\fP"
 Specify the udf revision to use. Valid revisions are 0x0201, 0x0200, 0x0150,
@@ -116,14 +116,14 @@
 .IP "\fB\-t \fIspeed\fP"
 Set write speed. (Defaults to 12x ?)
 .IP "\fB\-l \fItype\fP"
-Set multi\-session field. Either `0', `1', or `3', corresponding to
+Set multi\-session field. Either `0' (default), `1', or `3', corresponding to
 `No B0 pointer. Next Session not allowed',
 `B0 pointer = FF:FF:FF. Next session not allowed', and
 `Next session allowed. B0 pointer = next possible program area' respectively.
 .IP "\fB\-w \fImode\fP"
-Set write mode. Either `mode1' or `mode2'.
+Set write mode. Either `mode1' or `mode2' (default).
 .IP "\fB\-p \fItype\fP"
-Set packet type. Either `0' or `1', corresponding to
+Set packet type. Either `0' or `1' (default), corresponding to
 variable and fixed packet sizes respectively.
 .IP "\fB\-o \fIoffset\fP"
 Set write offset.
@@ -132,6 +132,7 @@
 .nf
 Jens Axboe <axboe@suse.de>
 Ben Fennema <bfennema@falcon.csc.calpoly.edu>
+Some additions by Richard Atterer <atterer@debian.org>
 .fi
 
 .SH AVAILABILITY
